괄호 처리 방식 통일, tokenize 메소드 통합

jangjichang opened the pull request:
리뷰를 읽고 반영했습니다

괄호 처리 방식을 통일했습니다.
tokenize 함수가 하는 작업을 정규식으로 하여 dart_to_hits 함수에 추가했습니다.

jangjichang committed:
괄호 처리 방식 통일, tokenize 메소드 통합

리뷰를 읽고 반영했습니다

괄호 처리 방식을 통일했습니다.
tokenize 함수가 하는 작업을 정규식으로 하여 dart_to_hits 함수에 추가했습니다.

ahastudio commented on dart-jjc-sk/dart_test.py:
```diff
@@ -2,38 +2,24 @@
     hits = []
     for token in tokens:
         m = re.search('([0-9]+)([SDT])(.?)', token)
         number, power, option = m.groups()
```
저라면 이 부분은 이렇게 할 것 같습니다.

```python
tokens = re.findall('([0-9]+)([SDT])([*#]?)', dart_result)
hits = []
for number, power, option in tokens:
```

jangjichang committed:
Add a group to regex

Add a group so that regular expressions looking for tokens return
number, power, or option
